apps/particl/particl.yml

```
@@ -1,7 +1,7 @@
 name: Particl
-description: 'A Decentralized Privacy Platform with a suite of tools to enhance your online privacy'
+description: 'World's Most Private & Secure Crypto Marketplace'
```

That apostrophe is breaking the yaml parser, which is why CI is failing for this PR.

```suggestion
description: 'World''s Most Private & Secure Crypto Marketplace'
```